brut.apktool/apktool-lib/src/main/java/brut/androlib/ApkDecoder.java

```diff
@@ -97,24 +93,21 @@ public void decode() throws AndrolibException, IOException, DirectoryException {
 
             LOGGER.info("Using Apktool " + Androlib.getVersion() + " on " + mApkFile.getName());
 
+            // recreate the resource table in case it was already created
+            mResTable = null;
```

Comment:
Is this for use in Apktool as a module? Can't follow why we'd want to empty what could have been already loaded in a situation.

Reply:
Oops. Not necessary, was part of something I was testing with minApiVersion, forgot to take it out before pushing. Committed a correction.